src/main/java/lotto/service/FacadeWinningServiceImpl.java

```diff
@@ -0,0 +1,111 @@
+package lotto.service;
+
+import static lotto.WinningCount.BONUS_COUNT;
+import static lotto.WinningCount.FIVE_COUNT;
+import static lotto.WinningCount.FOUR_COUNT;
+import static lotto.WinningCount.SIX_COUNT;
+import static lotto.WinningCount.THREE_COUNT;
+import static lotto.constants.LottoConstants.DEFAULT_WINNING_COUNT;
+import static lotto.constants.LottoConstants.PERCENTAGE_MULTIPLIER;
+import static lotto.constants.LottoConstants.PERCENTAGE_SCALE;
+import static lotto.constants.LottoConstants.WINNING_INCREMENT;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import lotto.model.entity.Lotto;
+import lotto.model.WinningRepository;
+import lotto.model.entity.LottoResult;
+
+public class FacadeWinningServiceImpl implements FacadeWinningService {
+
+    private final WinningRepository winningRepository;
+
+    public FacadeWinningServiceImpl(WinningRepository winningRepository) {
+        this.winningRepository = winningRepository;
+    }
+
+    @Override
+    public void saveWinningNumber(String winningNumbers) {
+        String[] winningNumbersStringList = winningNumbers.split(",");
+        List<Integer> winningNumberList = Arrays.stream(winningNumbersStringList)
+                .map(Integer::parseInt)
+                .collect(Collectors.toList());
+        winningRepository.saveWinningNumber(winningNumberList);
+    }
+
+    @Override
+    public void saveBonusNumber(int bonusNumber) {
+        winningRepository.saveBonusNumber(bonusNumber);
+    }
+
+    @Override
+    public void saveLottoResult(Map<Integer, Integer> matchedCountStatistics) {
```

Comment:
```suggestion
    public void saveLottoResult(MatchedCountStatistics matchedCountStatistics) {
```
일급컬렉션을 사용해봐요!

일급컬렉션은 언제 사용해야할까?도 같이 고민하면 좋을 것 같아요